python/common/python/time.java

```diff
@@ -53,8 +53,98 @@ public time() {
     }
 
     private static long vm_start_time;
-    public static org.python.Object _STRUCT_TM_ITEMS;
 
+    @org.python.Module(
+            __doc__ = "The tuple items are:\n" +
+                    "  year (including century, e.g. 1998)\n" +
+                    "  month (1-12)\n" +
+                    "  day (1-31)\n" +
+                    "  hours (0-23)\n" +
+                    "  minutes (0-59)\n" +
+                    "  seconds (0-59)\n" +
+                    "  weekday (0-6, Monday is 0)\n" +
+                    "  Julian day (day in the year, 1-366)\n" +
+                    "  DST (Daylight Savings Time) flag (-1, 0 or 1)\n" +
+                    "If the DST flag is 0, the time is given in the regular time zone;\n" +
+                    "if it is 1, the time is given in the DST time zone;\n" +
+                    "if it is -1, mktime() should guess based on the date and time.\n"
+    )
+    static class struct_time extends org.python.types.Module {
+        org.python.types.Tuple tuple;
+
+        public org.python.Object __getitem__(org.python.Object index) {
```

Comment:
It would be nice if this class also added support for accessing named attributes, like `time.struct_time` objects do in CPython:

```
>>> time.gmtime(0).tm_year
1970
>>> time.gmtime(0).tm_hour
0
>>> time.gmtime(0).tm_mday
1
```

Reply:
I will look into adding this feature.